modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/ObjectMapperUtil.java

```diff
@@ -32,6 +32,19 @@
  */
 public class ObjectMapperUtil {
 
+	public static <T> T readValue(Class<?> clazz, Object element) {
+		try {
+			return readValue(clazz, _objectMapper.writeValueAsString(element));
```

Comment:
I thought we could get read also of the try-catch but I see it is still necessary because of the writeValueAsString call...

Reply:
Yeah, me too :(